index.html

```diff
@@ -142,7 +144,8 @@ <h3>Here's a timeline of Dr. Favaloro's life:</h3>
     <li><strong>1986 - The Teacher Prize of Argentinian Medicine </strong></li>
     <li><strong>1987 - The Distinguished Alumnus Award of the Cleveland Clinic Foundation </strong></li>
     <li><strong>1987 - The Gairdner Foundation International Award (Toronto, Canada)</strong></li>
-    <li><strong>1989 - René Leriche Prize ,, granted by the International Surgery Society </strong></li>
```

Comment:
Kindly correct the double comma on your li

Reply:
- [ ] Tip: Please make at least two requested changes.

Reply:
@andym80 please it would be better if you give a well required changes like the first comment you gave  in order to test how good you are in reviewing